src/olympia/stats/management/commands/download_counts_from_file.py

```diff
@@ -31,14 +29,26 @@ def is_valid_source(src, fulls, prefixes):
 
 
 class Command(BaseCommand):
-    """Update download count metrics from a file in the database.
+    """Update download count metrics from stats_source in the database.
 
     Usage:
-    ./manage.py download_counts_from_file <folder> --date=YYYY-MM-DD
+    ./manage.py download_counts_from_file \
+        <folder> --date=YYYY-MM-DD --stats_source={s3,file}
 
     If no date is specified, the default is the day before.
-    If not folder is specified, the default is `hive_results/YYYY-MM-DD/`.
-    This folder will be located in `<settings.NETAPP_STORAGE>/tmp`.
+
+    If no stats_source is specified, the default is set to s3.
```

Comment:
Is the metrics server currently already copying files to S3? Or maybe both locations?

Reply:
Today the metrics server will upload the hive data to S3 and another cron job running on the admin host will copy the files to settings.NETAPP_STORAGE path.